src/components/CreateEmployeeForm.tsx

```diff
@@ -1,128 +1,184 @@
-    const dropdown3 = ['Product Engineering', 'Human Resource', 'Finance'];
+    const navigate = useNavigate();
+    const [file, setfiles] = useState(null);
+
+    const dropdown1 = [];
+    RoleData?.map(department => {
+        dropdown1.push({
+            'id': department.id,
+            'name': department.role
+        });
+    });
+    const dropdown2 = [];
+    DepartmentData?.map(department => {
+        dropdown2.push({
+            'id': department.id,
+            'name': department.name
+        });
+    });
+
 
     return (
-            <div className='m-6 mt-6 flex flex-initial  '>
-                <div className=' m-4 ml-[76px] h-[500px] w-[90%] rounded-xl bg-white  shadow-xl '>
-                    <form onSubmit={handleSubmit(() => { //data
-                        // console.log(data);
-                        reset();
-                    })}>
-                        <div className='flex p-2'>
-                            <div className='w-1/3 flex-initial '>
-                                <Label name='Employee Name' />
-                                <InputField registerFunction={register} placeholder='Employee Name'
-                                registerName='employeeName' type='string' />
-                                <p className='pl-6 font-sans text-xs normal-case 
-                                text-red-600'>{errors.employeeName?.message}</p>
-                            </div>
-                            <div className=' w-1/3 flex-initial '>
-                                <Label name='User Name' />
-                                <InputField registerFunction={register} placeholder='User Name' 
-                                registerName='userName' type='string' />
-                                <p className='pl-6 font-sans text-xs normal-case
-                                 text-red-600'>{errors.userName?.message}</p>
-                            </div>
-                            <div className=' w-1/3 flex-initial ' >
-                                <Label name='Age' />
-                                <InputField registerFunction={register} placeholder='Age' 
-                                registerName='age' type='number' />
-                                <p className='pl-6 font-sans text-xs normal-case 
-                                text-red-600'>{errors.age?.message}</p>
-                            </div>
+        <div className='mx-auto mt-6 flex flex-initial '>
+            <div className=' m-4 mx-auto h-[1200px] w-[55%] overflow-auto rounded-xl bg-white shadow-xl 
+            md:h-[1000px] md:w-[90%] md:justify-center lg:h-[650px] lg:w-[90%]'>
+                <form onSubmit={handleSubmit(async (SubmittedData) => {
+                        const addEmployeeResponse = await addEmployee(SubmittedData);
+                        if ('error' in addEmployeeResponse) {
+                            setErrorMessage(true);
+                        }
+                        else
```

Comment:
format the code before committing... just hit "Ctrl + shift + i" or set up a prettier precommit hook.

Reply:
this makes the code look neat and imporves readability